### DISCUSSION TITLE
how can i implement scroll animation by vanilla js?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#84750] [Programming Help] [answered]
### Body

i have some card now i want to implement scroll animation.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mhlehri** [comment, marked as answer]
![image](https://github.com/community/community/assets/132160334/2ac10f07-c964-4daf-84e7-52685568e854)

![image](https://github.com/community/community/assets/132160334/c074958a-d583-4672-b717-470a3cdd6646)

![image](https://github.com/community/community/assets/132160334/6b5f8c60-0034-48f7-b59e-79df594d04da)

you can try this way...
